src/datasources/staking-api/kiln-api.service.spec.ts

```diff
@@ -44,10 +44,8 @@ describe('KilnApi', () => {
   let stakingExpirationTimeInSeconds: number;
   let notFoundExpireTimeSeconds: number;
 
-  beforeEach(() => {
-    jest.resetAllMocks();
-
-    chainId = faker.string.numeric();
+  function createTarget(_chainId = faker.string.numeric()): void {
```

Comment:
Praise: I like the idea of creating new functions/methods to split the code and be DRY.